core/internal/cltest/cltest.go

```diff
@@ -1629,8 +1616,8 @@ func MustGetStateForKey(t testing.TB, kst keystore.Eth, key ethkey.KeyV2) ethkey
 	return states[0]
 }
 
-func NewTxmORM(t *testing.T, db *sqlx.DB, cfg pg.QConfig) txmgr.ORM {
-	return txmgr.NewORM(db, logger.TestLogger(t), cfg)
+func NewTxmStorageService(t *testing.T, db *sqlx.DB, cfg pg.QConfig) txmgr.EvmTxStorageService {
+	return txmgr.NewTxStorageService(db, logger.TestLogger(t), cfg)
```

Comment:
`StorageService` is very verbose. Does this say more than just `Storage`? `Data`? `Datastore`?
```suggestion
func NewTxmDatastore(t *testing.T, db *sqlx.DB, cfg pg.QConfig) txmgr.EvmTxDatastore {
	return txmgr.NewTxDatastore(db, logger.TestLogger(t), cfg)
```

Reply:
TxmStore?

Reply:
Yeah, I think that fits :heavy_plus_sign: